File: example-codes/speech-to-text.js
const formData = new FormData();
formData.append('api_key', 'YOUR_API_KEY');
formData.append('with_timestmap', 'false');
formData.append('with_diarization', 'false');
formData.append('data', audioFile, 'audio.wav');

const response = await fetch('https://paid-api.cantonese.ai', {
  method: 'POST',
  body: formData
});

const data = await response.json();
console.log(data);

File: example-codes/speech-to-text.py
import requests

url = "https://paid-api.cantonese.ai"

payload = {
    "api_key": "YOUR_API_KEY",
    "with_timestmap": "false",
    "with_diarization": "false"
}
files=[
  ('data',('audio.wav',open('ABSOLUTE_FILE_PATH','rb'),'audio/wav'))
]
headers = {}

response = requests.request("POST", url, headers=headers, data=payload, files=files)

print(response.text)